packages/components/src/tools-panel/tools-panel/README.md

```diff
@@ -157,6 +157,13 @@ wrapper element allowing the panel to lay them out accordingly.
 - Required: No
 - Default: `false`
 
+### `dropdownMenuProps`: `{}`
+
+The popover props to configure panel's `DropdownMenu`.
+
+-   Type: `Object`
```

Comment:
Let's write something like `DropdownMenuProps`, it's hopefully more informative than `Object`

Reply:
✅ https://github.com/WordPress/gutenberg/pull/55785/commits/10cfb5257c07bbf1f9064875237d17cd80d5ef7c